src/molecules/Modal/util/mobileInputScroll.js

```diff
@@ -0,0 +1,23 @@
+import elementDistanceFromTop from 'utils/elementDistanceFromTop';
+
+export default (dialogId) => {
+  if (navigator.maxTouchPoints) {
+    const scrollToInput = () => {
```

Comment:
Since you don't use `dialogId` in this function, I would pull this function out of the default function as a stand-alone function.